lib/event-subscribers.loader.ts

```diff
@@ -50,17 +50,16 @@ export class EventSubscribersLoader
         const { instance } = wrapper;
         const prototype = Object.getPrototypeOf(instance) || {};
         const isRequestScoped = !wrapper.isDependencyTreeStatic();
-        this.metadataScanner.scanFromPrototype(
-          instance,
-          prototype,
-          (methodKey: string) =>
+        this.metadataScanner
```

Comment:
Can you revert this change? This is a breaking one for those that are not on the latest version of @nestjs/core

Reply:
> This PR also included changes from #749
revert

Reply:
Ready.